packages/tools/client-debugger/client-debugger/src/test/ClientDebugger.test.ts

```diff
@@ -11,7 +11,7 @@ import { IFluidClientDebugger } from "../IFluidClientDebugger";
 import {
```

Comment:
I would recommend adding a test that uses the registry's listener pattern, and verifies that the listeners get called when we register / close debugger instances.

Reply:
Will appear after the unit tests PR merge into this one.